### PR TITLE
Update examples for the 2.2 specification

### PR DESCRIPTION
**Pull request by goneall**
Changes to the examples include:

- Renaming examples to 2.2
- Replacing deprecated licenses
- Updating spec version to 2.2
- Updating license version to 3.8
- Updating spreadsheet to new format

### CODE REVIEW

File: examples/SPDXTagExample-v2.2.spdx [hunk 1]
```diff
@@ -126,13 +135,11 @@ LicenseConcluded: LicenseRef-1
...
 LicenseComments: This license is used by Jena
 FileCopyrightText: <text>(c) Copyright 2000, 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009 Hewlett-Packard Development Company, LP</text>
-ArtifactOfProjectName: Jena
-ArtifactOfProjectHomePage: http://www.openjena.org/
 FileContributor: Hewlett Packard Inc.
 FileContributor: Apache Software Foundation
 FileDependency: ./lib-source/commons-lang3-3.1-sources.jar
 ## Relationships
-Relationship: SPDXRef-JenaLib DYNAMIC_LINK SPDXRef-Package
+Relationship: SPDXRef-JenaLib NOASSERTION SPDXRef-Package
```
Comment (kestewart) [inline examples/SPDXTagExample-v2.2.spdx:142]: Relationship of NOASSERTION doesn't make sense here.

File: examples/SPDXXMLExample-v2.2.spdx.xml [hunk 2]
```diff
@@ -409,23 +411,23 @@ THE SOFTWARE IS PROVIDED �AS IS', WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMP
...
-    <relatedSpdxElement>SPDXRef-JenaLib</relatedSpdxElement>
-    <relationshipType>CONTAINS</relationshipType>
+    <relatedSpdxElement>SPDXRef-Saxon</relatedSpdxElement>
+    <relationshipType>DYNAMIC_LINK</relationshipType>
   </relationships>
   <relationships>
     <spdxElementId>SPDXRef-JenaLib</spdxElementId>
     <relatedSpdxElement>SPDXRef-Package</relatedSpdxElement>
-    <relationshipType>DYNAMIC_LINK</relationshipType>
+    <relationshipType>NOASSERTION</relationshipType>
```
Comment (kestewart) [inline examples/SPDXXMLExample-v2.2.spdx.xml:430]: Check the relationship here.

**Comment by kestewart** [comment]
@goneall - can you please double check the relationship I flagged.    NOASSERTION is being used in wrong place I believe.   see example:
Relationship: SPDXRef-BobBrowser CONTAINS NOASSERTION

File: examples/SPDXXMLExample-v2.2.spdx.xml (hunk 2 above)
Comment (goneall) [reply examples/SPDXXMLExample-v2.2.spdx.xml]: The examples have been updated - this should now be resolved.

**Pull request merged**